content/en/cloud_cost_management/anomalies.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Anomalies Page
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/monitors"
+  tag: "Monitors"
+  text: "Create Cost Monitors"
+---
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) continuously monitors your environment to identify and prioritize unexpected cost changes, so that you can detect, share, investigate, and resolve cost anomalies. Cost anomalies are currently available for AWS, and does not require any additional set up after CCM is set up.
+
+## View cost anomalies
+
+You can filter to Active, Past, or Resolved anomalies:
+- **Active**: An anomaly is **Active** if there were anomalous costs in the last day (the last full day of cost data, which is often 2-3 days prior). 
+- **Past**: If an anomaly lasts more than 7 days, or the algorithm detects that costs are no longer anomalous, the anomaly is moved to the **Past** tab. Past anomalies can be useful to report on, but are often less urgent and actionable.
+- **Resolved**: If you Resolve an anomaly, the anomaly can be found in the **Resolved** tab.
+
+Each anomaly explains how much more costs were than expected, the specific cloud service name (ex: rds), usage type (ex: db.r6g.12xl), and cloud accounts where the anomaly occured. The anomaly also shows what expected costs would have been in the time frame, with a graph that shows the cost pattern over the past 1 month.
```

Comment:
It seems like this explanation of what an anomaly is could be moved up into the overview, before the page gets into the details of filtering anomalies etc.?

Reply:
good point - i've moved it up